homeassistant/components/blebox/config_flow.py

```diff
@@ -0,0 +1,128 @@
+"""Config flow for BleBox devices integration."""
+import logging
+
+from blebox_uniapi.error import Error, UnsupportedBoxVersion
+from blebox_uniapi.products import Products
+from blebox_uniapi.session import ApiHost
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import (
+    ALREADY_CONFIGURED,
+    CANNOT_CONNECT,
+    DEFAULT_SETUP_TIMEOUT,
+    DOMAIN,
+    UNKNOWN,
+    UNSUPPORTED_VERSION,
+)
+
+DEFAULT_HOST = "192.168.0.2"
+DEFAULT_PORT = 80
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def host_port(data):
+    """Return a list with host and port."""
+    return (data[CONF_HOST], data[CONF_PORT])
+
+
+def create_schema(previous_input=None):
+    """Create a schema with given values as default."""
+    if previous_input is not None:
+        host, port = host_port(previous_input)
+    else:
+        host = DEFAULT_HOST
+        port = DEFAULT_PORT
+
+    return vol.Schema(
+        {
+            vol.Required(CONF_HOST, default=host): str,
+            vol.Required(CONF_PORT, default=port): int,
+        }
+    )
+
+
+LOG_MSG = {
+    UNSUPPORTED_VERSION: "Outdated firmware",
+    CANNOT_CONNECT: "Failed to identify device",
+    UNKNOWN: "Unknown error while identifying device",
+}
+
+
+class BleBoxConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for BleBox devices."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialize the BleBox config flow."""
+        self.device_config = {}
+
+    def handle_step_exception(self, step, exception, schema, addr, message_id):
+        """Handle step exceptions."""
+
+        _LOGGER.error("%s at %s:%d (%s)", LOG_MSG[message_id], *addr, exception)
```

Comment:
```suggestion
        _LOGGER.debug("%s at %s:%d (%s)", LOG_MSG[message_id], *addr, exception)
```

Reply:
If we know what the exception is we shouldn't log it.  Probably ok for unknown.

Reply:
As suggested, except a warning is shown for connection errors. (These can be common and hard to diagnose, while the translated message explains to "see the logs", and these errors are too important - for now - to expect users to enable debug logs).